test/task.js

```diff
@@ -45,6 +46,83 @@ test('browserify handles unknown input', async t => {
 	}
 });
 
+test.serial('check babelconfig output', async t => {
+	t.plan(1);
+	fs.writeFileSync('test/.babelrc', `{
+	"presets": [
+		"@babel/preset-env"
+	],
+	"plugins": ["@babel/plugin-proposal-object-rest-spread"]
+}`);
+	const input = 'let { x, y, ...z } = { x: 1, y: 2, a: 3, b: 4 };';
+	const expected = `"use strict";
+
+function _objectWithoutProperties(source, excluded) { if (source == null) return {}; var target = _objectWithoutPropertiesLoose(source, excluded); var key, i; if (Object.getOwnPropertySymbols) { var sourceSymbolKeys = Object.getOwnPropertySymbols(source); for (i = 0; i < sourceSymbolKeys.length; i++) { key = sourceSymbolKeys[i]; if (excluded.indexOf(key) >= 0) continue; if (!Object.prototype.propertyIsEnumerable.call(source, key)) continue; target[key] = source[key]; } } return target; }
+
+function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }
+
+var _x$y$a$b = {
+  x: 1,
+  y: 2,
+  a: 3,
+  b: 4
+},
+    x = _x$y$a$b.x,
+    y = _x$y$a$b.y,
+    z = _objectWithoutProperties(_x$y$a$b, ["x", "y"]);`;
+
+	const code = await task.runBabel(input, {babelConfig: 'test/.babelrc'});
```

Comment:
Replacing `{babelConfig: 'test/.babelrc'}` with `{}` gives the same output, so I guess this test is useless